rpcd/playbooks/roles/rpc_maas/tasks/package_install.yml

```diff
@@ -26,6 +26,4 @@
     name: "{{ item }}"
     state: present
     extra_args: "{{ pip_install_options | default('') }}"
-  with_flattened:
-    - maas_pip_packages
-    - maas_pip_dependencies
+  with_items: maas_pip_packages + maas_pip_dependencies
```

Comment:
Please read the docs. The documentation shows that the proper way to do this is with what I posted. Granted it's more necessary for nested lists but my example should be the correct way of doing this and we were still seeing the swift config dict being interpolated into the dependencies even with your change (based on what Dave and I saw in the logs).